packages/celotool/src/cmds/account/invite.ts

```diff
@@ -48,7 +50,7 @@ export const handler = async (argv: InviteArgv) => {
     // TODO: this default gas price might not be accurate
     const gasPrice = 100000000000
 
-    const temporaryWalletAccount = await kit.web3.eth.accounts.create()
+    const temporaryWalletAccount = web3.eth.accounts.create()
```

Comment:
This is an account creation using web3. We don't have any of ours wallet that make that. So in a way, if the user set a wallet (the azure-hsm, for example), I don't know if it will be clear for the user, that we are just calling a pub/priv key creation, instead of the wallet set.  
I agree that is something that we could eventually add, but as it was never part of our kit, I prefer to leave it to a new feature/PR